examples/complex_type.cpp

```diff
@@ -83,6 +83,14 @@ int main()
     std::cout << "Enumeration::C: " << value << std::endl;
     std::cout << "Enumeration2::C: " << value2 << std::endl;
 
+    std::cout << "-----------------------" << std::endl;
+    submod_b.enum_32(std::move(my_enum));
+     root.create_constant("MyConstEnum", enum_data); // TODO: cast enumerations
+    DynamicData my_const(primitive_type<uint64_t>());
+    my_const = 555ul;
+    root.create_constant("MyConst", my_const);
```

Comment:
Is this possible to add something like
```c++
root.create_constant("MyConst", 555ul);
```
?

Reply:
A constant could be of any primitive type. Do you mean create the DynamicData within a templated `create_constant` method? I don't know if it is really useful...

Reply:
Yeah, this is exactly what I mean. It can save the user 2 lines more :D

Reply:
As it is unrelated to this PR, it should be a new task, a new PR. B-)

Reply:
Done